lib/mj-single.js

```diff
@@ -63,6 +61,7 @@ var defaults = {
   img: false,                     // return img tag for remote image?
   png: false,                     // return png image (as data: URL)?
   dpi: 144,                       // dpi for png image
+  scale: 2,                       // scale for png image
```

Comment:
Why is this 2 by default?  Wouldn't a normal default scaling be 1?  And how does this relate to the dpi value?  Shouldn't scaling be determined by the dpi and ex values?